### DISCUSSION TITLE
developer tools

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#314] [Help] [answered]
if i already have @griffpatch developer tools already installed on chrome and i enable the developer tools in the add-on setting
will it break?

**Comment by jeffalo** [comment]
i'm also wondering how it currently handles that too.

can browser extensions see other installed extensions? it would be neat if addons could specify the id of the standalone versions of themselves so that scratch addons handles it properly

**Comment by WorldLanguages** [comment, marked as answer]
The extension takes priority and the addon won't run.

> can browser extensions see other installed extensions?

I believe it needs some permissions, and even if it didn't I don't think it would be worth it.

**Comment by scratchusernamemrtbts** [comment]
thanks
